tgui/packages/tgui-panel/styles/components/Chat.scss

```diff
@@ -82,7 +83,7 @@ $color-bg-section: base.$color-bg-section !default;
 
 .ChatMessage--highlighted {
   position: relative;
-  border-left: (1em / 6) solid rgba(255, 221, 68);
+  border-bottom: math.div(1em, 6) solid colors.$red;
```

Comment:
This is why Highlighted messages have a red line under them. Previously this added a Yellow border to the left, now it adds a Red Border to the bottom

Reply:
I have no idea why would you risk your sanity and review the 71 file PR, but neat. Gonna change that

Reply:
I can not overstate how much I'm supposed to. Also this is kind of HTML? Sort of. I'm somewhat familiar with it